src/renderer/components/+workloads-overview/overview.tsx

```diff
@@ -24,7 +23,6 @@ interface Props extends RouteComponentProps<IWorkloadsOverviewRouteParams> {
 
 @observer
 export class WorkloadsOverview extends React.Component<Props> {
-  @observable isReady = false;
   @observable isUnmounting = false;
```

Comment:
I think that this is the wrong way to have this, it should be a list of disposers which is called in the `componentWillUnmount` function.

It is useful to still have this variable but I don't think it should be observable since it is only read from `componentDidMount` and only written to in `componentWillUnmount`.

Reply:
The problem with `componentWillUnmount` that it might not work as expected cause of async `componentDidMount`..
So this is better solution in that case..